Dockerfile

```diff
@@ -39,7 +39,8 @@ RUN set -ex; \
       libminiupnpc-dev \
       libnatpmp-dev \
       libpsl-dev \
-      libssl-dev
+      libssl-dev \
+      natpmpc
```

Comment:
why was natpmpc added?

Reply:
Apologies. This was not intended. Can this commit be deleted?